apps/src/templates/ProjectTemplateWorkspaceIcon.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import ReactTooltip from 'react-tooltip';
+import _ from 'lodash';
+var msg = require('@cdo/locale');
+
+const IMAGE_BASE_URL = '/blockly/media/';
+
+const styles = {
+  container: {
+    display: 'inline',
+  },
+  tooltip: {
+    maxWidth: 200,
+    lineHeight: '20px',
+    whiteSpace: 'normal',
+  },
+  projectTemplateIcon: {
+    marginRight: 5,
+    marginTop: -1,
+  },
+};
+
+export default class ProjectTemplateWorkspaceIcon extends React.Component {
+
+  render() {
+    const tooltipId = _.uniqueId();
```

Comment:
I'm not sure it matters, but I wonder if it's more correct to create a unique ID when the component is constructed, instead of every time it's rendered?

```jsx
constructor(props) {
  super(props);
  this.tooltipId = _.uniqueId();
}
```

Reply:
I copied how `TooltipWithIcon` is doing it. That does seem like a reasonable idea though. I'll try it out...